tests/test_product.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from buy_together_app.models import Product
+
+QR = "Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+NAME = "Banana"
+DESCRIPTION = "Good!"
+
+
+@pytest.fixture
+def product0():
+    return Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
+
+
+@pytest.fixture
+def saved_product0(product0):
+    product0.save()
+    return product0
+
+
+test_data = [
```

Comment:
If you want it as a constant, then the convention is to have it in uppercase.

Having said that, i suggest making it a fixture instead. it will make sure that even if one test is changing it for some reason, the next test case will get a fresh copy as the fixture is recreated.

Reply:
Done,cant use it as fixture because i have imported it in the migrations tests